boot/bootutil/src/loader.c

```diff
@@ -1726,33 +1764,34 @@ boot_go(struct boot_rsp *rsp)
         }
         /* Since headers were reloaded, it can be assumed we just performed a
          * swap or overwrite. Now the header info that should be used to
-         * provide the data for the bootstrap, which previously was at Slot 1,
-         * was updated to Slot 0.
+         * provide the data for the bootstrap, which previously was at secondary
+         * slot, was updated to primary slot.
          */
-        slot = 0;
+        slot = BOOT_PRIMARY_SLOT;
     }
 
-#ifdef MCUBOOT_VALIDATE_SLOT0
-    rc = boot_validate_slot(0, NULL);
+#ifdef mcuboot_validate_primary_slot
```

Comment:
```suggestion
MCUBOOT_VALIDATE_PRIMARY_SLOT
```

@davidvincze you can do
```
   ```suggestion
    <change>
    ```
```
In github now, it's a nice feature for one liner fixes